Pj4all

andreiavram opened the pull request.

florinapauleti committed:
Un singur tip de CL (filială) 

Modificări privind existența de Centre locale doar cu personalitate juridică, adică filiale, față de împărțirea anterioară în filiale și sucursale.

florinapauleti committed:
Un singur tip de CL (filială)

Modificări aduse pentru ca toate CL să fie cu personalitate juridică, adică filiale, și să nu mai existe sucursale.

florinapauleti committed:
IEDLOC

Introducerea conceptului IEDLOC în regulament și a anexei IEDLOC

florinapauleti committed:
Anexa IEDLOC

Anexa care cuprinde toate nivelurile de dezvoltare IEDLOC cu criteriile și responsabilitățile CL cât și atribuțiile ONCR pentru fiecare nivel. Conține și când se face evaluarea CL.

florinapauleti committed:
Înființare / Desființare de CL 

Am agăugat că înființarea sau desființarea se face în baza hotărârii adunării generale

florinapauleti committed:
Înființare / divizare / desființare de CL

florinapauleti committed:
Doar filiale la nivel de ONCR

florinapauleti committed:
înființare CL 

minim de membri pt CL și disparitia ideii de conditii pentru acordare de PJ din atributiile EE

florinapauleti committed:
Drepturi și obligații CL

întărirea conceptului de Centre locale legal înființate, adica care sunt deja inscrise in registrul asociatiilor si fundatiilor, cât și drepturile si obligatiile tuturor CL

chabinush committed:
Update statut_pregatit_feedback.txt

ștergere act constitutiv

chabinush committed:
Update regulament_pregatit_feedback.txt

ștergere act constitutiv

andreiavram commented on regulament_pregatit_feedback.txt:
```diff
@@ -535,54 +541,30 @@ Art. 87. Procedura de înființare a unui Centru Local este următoarea:
+		b. să fi îndeplinit cel puțin criteriile minime pentru Nivelul 1 - Centre Locale în dezvoltare, în evaluarea IEDLOC.
+		c. în cazul Grupurilor externe care cer să devină Centre Locale este necesară recomandarea din partea Centrului Local Mentor.
+	(3) Centrele Locale astfel înființate au personalitate juridică. Filiala se constituie prin hotărârea Adunării Generale a ONCR. Personalitatea juridică se dobândește de la data înscrierii filialei în Registrul asociațiilor și fundațiilor.
+	(4) Toate Centrele Locale la înființare semnează un contract cu ONCR, prin care primesc susținerea organizației pentru a-și desfășura activitatea și a reprezenta local ONCR, drepturi care pot fi suspendate de Consiliul Director și retrase de Adunarea Generală.
```
oare n-ar fi bine să intrăm mai în detaliu cu contractul ăsta?

florinapauleti replied:
La ce te gândești când zici că mai în detaliu? să menționăm în regulament ce ar trebui să conțină contractul? Eu mă gândeam că venim cu o anexa cu model standard de contract.

andreiavram replied:
N-aș pune contractul ca anexă la Regulament, că asta complică mult schimbarea lui. Mai repede aș lista cumva elementele minime pe care contractul ar trebui să le conțină, și eventual lucrurile pe care contractul nu are voie să le conțină, fără să limitez prea mult capacitatea de reacție a CD la realitățile cu care o să se confrunte going forward.

chabinush replied:
Contractul: nume și logo-ul entității, drepturile și obligațiile (dreptul de a participa la activități, obligația de plată a cotizației etc), reprezentarea cercetășiei (misiunea, viziune, valori și branding/reguli etc), angajament față de formare, angajament față de dezvoltarea comunității, respectarea Regulamentului, Statutului și deciziilor AG și CD, condițiile în care contractul este anulat (rezilierea contractului) etc

chabinush replied:
Contractul (charter-ul, mandatul) nu ar trebui să se lege de persoane (nu apar nume de oameni, ci se leagă de structură);

florinapauleti replied:
done

florinapauleti committed:
Update contract CL cu ONCR

detalii privind conținutul contractului cadru dintre CL și ONCR de la înființare

chabinush commented on regulament_pregatit_feedback.txt:
```diff
@@ -535,54 +541,31 @@ Art. 87. Procedura de înființare a unui Centru Local este următoarea:
+		c. în cazul Grupurilor externe care cer să devină Centre Locale este necesară recomandarea din partea Centrului Local Mentor.
+	(3) Centrele Locale astfel înființate au personalitate juridică. Filiala se constituie prin hotărârea Adunării Generale a ONCR. Personalitatea juridică se dobândește de la data înscrierii filialei în Registrul asociațiilor și fundațiilor.
+	(4) Toate Centrele Locale la înființare semnează un contract cu ONCR, prin reprezentantul legal, în numele entității, prin care primesc susținerea organizației pentru a-și desfășura activitatea și a reprezenta local ONCR, drepturi care pot fi suspendate de Consiliul Director și retrase de Adunarea Generală.
+	(5) Formatul contractului este decis de Consiliul Director și conține minimum următoarele elemente: numele și logo-ul entității, drepturile și obligațiile Centrelui Local, informații despre reprezentarea cercetășiei (misiunea, viziune, valori, reguli de brand etc.), angajamentul față de formare, angajamentul față de dezvoltarea comunității, asumarea respectării Regulamentului, Statutului și a de ...
```
Alineatul (5): ”implicare în comunitate” în loc de ”dezvoltarea comunității”

chabinush commented on regulament_pregatit_feedback.txt:
```diff
@@ -535,54 +541,31 @@ Art. 87. Procedura de înființare a unui Centru Local este următoarea:
-	e. dreptul de a deschide cont bancar propriu: Centrele Locale cu personalitate juridică (filialele); Centrele Locale fără personalitate juridică (sucursalele) pot solicita girul unei filiale sau al ONCR.
-	f. obligația de a depune situația financiară anuală: toate Centrele Locale, filialele conform art. 88, iar sucursalele la trezorierul ONCR.
-	g. obligația de a depune alte raporturi financiare: toate Centrele Locale, conform art. 89.
+	e. dreptul de a deschide cont bancar propriu: Centrele Locale legal înființate.
```
Art 95 , alin e (linia 562) - deschiderea de conturi nu va mai ține de ONCR, deci ar trebui ștearsă

chabinush commented on regulament_pregatit_feedback.txt:
```diff
@@ -497,13 +496,20 @@ Art. 80. Condițiile minime pe care trebuie să le îndeplinească un lider adul
 	În cazul în care se solicită înființarea unui Grup Candidat ca urmare a retragerii din Centrul Local existent a unor lideri și cercetași, aceasta nu va fi pusă în discuție decât după cercetarea de către Consiliul Director sau comisia desemnată de acesta a împrejurărilor care au determinat această situație.
 
-	În cazul în care un Centru Local cu personalitate juridică, ca urmare a hotărârii Adunării Generale a Centrului Local se divizează, grupul nou este propus Adunării Generale a ONCR pentru constituire direct ca Centru Local cu personalitate juridică, condiția necesară fiind îndeplinirea criteriilor și procedurilor statutare pentru aceasta forma de organizare.
+	(1) În cazul în care un Centru Local, ca urmare a hotărârii Adunării Generale a Centrului Local se divizează, grupul nou este propus Adunării Generale a ONCR pentru constituire, condiția necesară fiind îndeplinirea criteriilor și procedurilor statutare pentru aceasta formă de organizare.
```
de reformulat cât să aibă chiar sens propoziția (de ex: scoaterea textului de la final ”pentru această formă de organizare” sau menționăm constituire ca CL)

chabinush replied:
feedback Ela: să lăsăm până la ”AG a ONCR pentru constituire” - și să ștergem tot textul de după.

florinapauleti committed:
Update art 88

implicare in comunitate în loc de dezvoltarea comunității

florinapauleti committed:
last updates.

after feedback

andreiavram committed:
Merge branch 'master' into pj4all
